usdoj-crt/beta-ada
# File a complaint icons

Changes:

1.  Added icons to the File a Complaint page - this required both changes to the markdown (it's now markup) and styling. 
2.  Added the icon files and changed the fill colors to match the design.

To test:

1.  Check that icons align as per the design on all screen sizes: mobile, tablet and desktop
2.  Make sure that the styling on both the translation pages is identical.
